python/res/enkf/jobs/enkf_scaling_job_config.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
+import configsuite
+import six
+
+from copy import deepcopy
+
+from configsuite import MetaKeys as MK
+from configsuite import types
+
+
+@configsuite.validator_msg("Minimum length of index list must be > 1 for PCA")
+def _min_length(value):
+    if len(value) > 1:
+        return True
+    return False
+
+
+@configsuite.validator_msg("Minimum value of index must be >= 0")
+def _min_value(value):
+    if value >= 0:
+        return True
+    return False
+
+
+_num_convert_msg = "Will go through the input and try to convert to list of int"
+@configsuite.transformation_msg(_num_convert_msg)
+def _to_int_list(value):
+    value = deepcopy(value)
+    if isinstance(value, six.integer_types):
+        return [value]
+    elif isinstance(value, (list, tuple)):
+        value = ",".join([str(x) for x in value])
+    return _realize_list(value)
+
+
+@configsuite.transformation_msg("Convert ranges and singeltons into list")
+def _realize_list(l):
```

Comment:
single letter parameters should be avoided

Reply:
:+1: